commands/bmc.vegman

```diff
@@ -116,12 +116,17 @@ function cmd_datetime_method {
   method=${method##*.}
   method=${method%\"*}
 
+  # VEGMAN doesn't have RTC and in manual mode it uses PCH time.
+  if [ "${method}" = "Manual" ]; then
+      method="HostTime"
+  fi
+
   if [[ $# -eq 0 ]]; then
       echo "Current time synchronization method is ${method}"
   else
     case "${1}" in
-      [Mm][Aa][Nn][Uu][Aa][Ll])
-        if [[ ${method} != "Manual" ]]; then
+      [Hh][Oo][Ss][Tt][Tt][Ii][Mm][Ee])
+        if [[ ${method} != "HostTime" ]]; then
```

Comment:
That's look strange: why do you need to translate dbus property value from "Manual" to "HostTime" and then back. I get it as you want to change CLI command argument, but this doesn't require to also change property value.
From first look this "if [[ ${method} != "HostTime" ]]; then ... set ... Method.Manual" looks as error and it took me some time to understand what's going on here.

Reply:
API changing affects to many other software (settings, phosphor-time-manager, bmcweb, webui and so on). So, this masquerade seems to me a little bit easier to implement.

Reply:
Oh, and the property value is displayed to user on line 125.

Reply:
ok

Reply:
But you can have this masquerading 'if' between lines 124 and 125

Reply:
I have got what do you meant... likely... 
fixed :)